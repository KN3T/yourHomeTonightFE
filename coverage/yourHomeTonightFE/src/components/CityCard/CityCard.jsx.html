
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for yourHomeTonightFE/src/components/CityCard/CityCard.jsx</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../../../../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../../../../index.html">All files</a> / <a href="index.html">yourHomeTonightFE/src/components/CityCard</a> CityCard.jsx</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>27/27</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>1/1</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>1/1</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>27/27</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
        <template id="filterTemplate">
            <div class="quiet">
                Filter:
                <input oninput="onInput()" type="search" id="fileSearch">
            </div>
        </template>
    </div>
    <div class='status-line high'></div>
    <pre><table class="coverage">
<tr><td class="line-count quiet"><a name='L1'></a><a href='#L1'>1</a>
<a name='L2'></a><a href='#L2'>2</a>
<a name='L3'></a><a href='#L3'>3</a>
<a name='L4'></a><a href='#L4'>4</a>
<a name='L5'></a><a href='#L5'>5</a>
<a name='L6'></a><a href='#L6'>6</a>
<a name='L7'></a><a href='#L7'>7</a>
<a name='L8'></a><a href='#L8'>8</a>
<a name='L9'></a><a href='#L9'>9</a>
<a name='L10'></a><a href='#L10'>10</a>
<a name='L11'></a><a href='#L11'>11</a>
<a name='L12'></a><a href='#L12'>12</a>
<a name='L13'></a><a href='#L13'>13</a>
<a name='L14'></a><a href='#L14'>14</a>
<a name='L15'></a><a href='#L15'>15</a>
<a name='L16'></a><a href='#L16'>16</a>
<a name='L17'></a><a href='#L17'>17</a>
<a name='L18'></a><a href='#L18'>18</a>
<a name='L19'></a><a href='#L19'>19</a>
<a name='L20'></a><a href='#L20'>20</a>
<a name='L21'></a><a href='#L21'>21</a>
<a name='L22'></a><a href='#L22'>22</a>
<a name='L23'></a><a href='#L23'>23</a>
<a name='L24'></a><a href='#L24'>24</a>
<a name='L25'></a><a href='#L25'>25</a>
<a name='L26'></a><a href='#L26'>26</a>
<a name='L27'></a><a href='#L27'>27</a>
<a name='L28'></a><a href='#L28'>28</a></td><td class="line-coverage quiet"><span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react';
&nbsp;
import './CityCard.scss';
&nbsp;
const CityCard = () =&gt; {
  return (
    &lt;div&gt;
      &lt;div className="city_card"&gt;
        &lt;div className="city_card_wrapper"&gt;
          &lt;div className="city_card_img_wrapper"&gt;
            &lt;img
              className="city_card_img"
              src="https://cf.bstatic.com/xdata/images/hotel/max1280x900/328270670.jpg?k=e1d9d748699b1aa3257fb190da472def6689647b01a3c540e219fab6b771c184&amp;o=&amp;hp=1"
              alt="city_card_img"
            /&gt;
          &lt;/div&gt;
          &lt;div className="city_card_info"&gt;
            &lt;div className="city_name"&gt;City Name&lt;/div&gt;
            &lt;div className="city_total"&gt;13,133 motels&lt;/div&gt;
          &lt;/div&gt;
        &lt;/div&gt;
      &lt;/div&gt;
    &lt;/div&gt;
  );
};
&nbsp;
export default CityCard;
&nbsp;</pre></td></tr></table></pre>

                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank" rel="noopener noreferrer">istanbul</a>
                at Sat Jul 09 2022 15:08:23 GMT+0700 (Indochina Time)
            </div>
        <script src="../../../../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../../../../sorter.js"></script>
        <script src="../../../../block-navigation.js"></script>
    </body>
</html>
    