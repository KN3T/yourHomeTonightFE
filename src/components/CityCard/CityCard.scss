.city_card {
  width: 100%;
  overflow: hidden;
  display: flex;
  flex-direction: column;
  align-items: center;
  &_img_wrapper {
    height: 15rem;
    .city_card_img {
      height: 100%;
      object-fit: cover;
      border-radius: 5px;
    }
  }
  &_info {
    width: 100%;
    .city_name {
      padding-top: 5px;
      font-size: 15px;
      font-weight: 500;
    }
    .city_total {
      color: #777777;
      font-size: 14px;
    }
  }
}

@media screen and (max-width: 777px) {
  .city_card {
    &_img_wrapper {
      height: 15rem;
      .city_card_img {
        width: 18rem;
      }
    }
  }
}

@media screen and (max-width: 645px) {
  .city_card {
    &_img_wrapper {
      height: 15rem;
      .city_card_img {
        width: 15rem;
      }
    }
  }
}

@media screen and (max-width: 557px) {
  .city_card {
    &_img_wrapper {
      height: 15rem;
      .city_card_img {
        width: 13rem;
      }
    }
  }
}
