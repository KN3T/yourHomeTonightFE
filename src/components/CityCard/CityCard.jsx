import React from 'react';

// import { Typography } from 'antd'
import './CityCard.scss';

// const { Title } = Typography;
const CityCard = () => {
  return (
    <div>
      {/* <Title level={2}>Our Most Populer City</Title> */}
      <div className="city_card">
        <div className="city_card_wrapper">
          <div className="city_card_img_wrapper">
            <img
              className="city_card_img"
              src="https://cf.bstatic.com/xdata/images/hotel/max1280x900/328270670.jpg?k=e1d9d748699b1aa3257fb190da472def6689647b01a3c540e219fab6b771c184&o=&hp=1"
              alt="city_card_img"
            />
          </div>
          <div className="city_card_info">
            <div className="city_name">City Name</div>
            <div className="city_total">13,133 motels</div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default CityCard;
