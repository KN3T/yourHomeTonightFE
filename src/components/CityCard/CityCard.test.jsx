import { render } from '@testing-library/react';

import CityCard from './CityCard';

test('city card', () => {
  const component = render(<CityCard />);
  expect(component).toMatchSnapshot();
});
