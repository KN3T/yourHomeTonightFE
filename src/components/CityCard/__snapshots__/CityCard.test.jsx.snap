// Vitest Snapshot v1

exports[`city card 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="city_card"
        >
          <div
            class="city_card_wrapper"
          >
            <div
              class="city_card_img_wrapper"
            >
              <img
                alt="city_card_img"
                class="city_card_img"
                src="https://cf.bstatic.com/xdata/images/hotel/max1280x900/328270670.jpg?k=e1d9d748699b1aa3257fb190da472def6689647b01a3c540e219fab6b771c184&o=&hp=1"
              />
            </div>
            <div
              class="city_card_info"
            >
              <div
                class="city_name"
              >
                City Name
              </div>
              <div
                class="city_total"
              >
                13,133 motels
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="city_card"
      >
        <div
          class="city_card_wrapper"
        >
          <div
            class="city_card_img_wrapper"
          >
            <img
              alt="city_card_img"
              class="city_card_img"
              src="https://cf.bstatic.com/xdata/images/hotel/max1280x900/328270670.jpg?k=e1d9d748699b1aa3257fb190da472def6689647b01a3c540e219fab6b771c184&o=&hp=1"
            />
          </div>
          <div
            class="city_card_info"
          >
            <div
              class="city_name"
            >
              City Name
            </div>
            <div
              class="city_total"
            >
              13,133 motels
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
